sqlserver/assets/configuration/spec.yaml

```diff
@@ -713,6 +713,14 @@ files:
         type: number
         example: 1800
         display_default: false
+    - name: schemas_collection_interval 
```

Comment:
How can we disable schema collection? We have to assume that the collection can cause problems on some databases.

Reply:
+1 this feature should be disabled by default as well

Reply:
good point, adding enabled and putting under `if self._config.dbm_enabled:`

Reply:
Not a must have for this PR, but would it generally make sense to introduce separate `enable` switches for `partitions` and `columns`? That would be for the case when e.g. a high number of partitions is causing a performance issue. Customer would in this case till benefit from table collections.